docs/workflow/debugging/mono/android-debugging.md

```diff
@@ -0,0 +1,76 @@
+# Debugging Android runtime issues
+
+## Enable verbose runtime logging
+
+If a `net6.0-android` (or later) C# project is available, adding the following:
+
+```xml
+ <ItemGroup>
+   <AndroidEnvironment Include="AndroidEnv.txt" />
+ </ItemGroup>
+```
+
+with `AndroidEnv.txt`:
+
+```
+debug.mono.log=mono_log_level=debug,mono_log_mask=all
+MONO_SDB_ENV_OPTIONS=loglevel=10    # only needed if you're debugging the managed debugger
+```
+
+Will enable additional Mono runtime logging in the adb log.  This is often
+enough to diagnose issues such as missing assemblies or other loader problems.
+
+## Managed Debugging
+
+Should work from Visual Studio (Windows)
+
+## Native debugging
+
+Install Android Studio.
+
+Download the symbols nupkg corresponding to the runtime pack that is used by
+the app.  The runtime pack for the android workload is in a folder like
+`${DOTNET_ROOT}/packs/Microsoft.NETCore.App.Runtime.Mono.android-x86/6.0.0-rc.1.21451.13`
+
+The symbols are in a package called
+`Microsoft.NETCore.App.Runtime.Mono.android-x86.6.0.0-rc.1.21451.13.symbols.nupkg`
+uploaded to (FIXME: where does the symbols nuget go?).  Extract it to some folder using `unzip` and in the
```

Comment:
I don't know how to do this without digging through AzDO pipeline control panels.

Is there a nice method to download a symbols .nupkg for a given runtime pack? /cc @akoeplinger @steveisok

Reply:
/cc @directhex